src/services/liveCarsService.js

```
@@ -0,0 +1,10 @@
+import baseService from './baseService';
+
+export const getLiveCars = async () => {
+    try {
+      const response = await baseService.get(`local-cars/get-all-local-cars?type=live`);
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)